src/depackers/oxm.c

```diff
@@ -20,7 +20,7 @@
  * THE SOFTWARE.
  */
 
-#include "vorbis.h"
+#include "stb_vorbis.h"
```

Comment:
IMO, we should keep the name as it is w/o `stb_` prefix.

Will look at the rest later.

Reply:
I second leaving the filenames the same for now. This would simplify the patch a little, and this file may end up getting renamed/moved to `src/loaders/` in the near future anyway (Claudio greenlit merging the OXM depacker into the sample code today in email).